lib/windows/plugin/containers/FirmwareDialogContainer.js

```diff
@@ -52,7 +53,10 @@ function mapStateToProps(state) {
 function mapDispatchToProps(dispatch) {
     return {
         onConfirmUpdateFirmware: port => dispatch(FirmwareDialogActions.updateFirmware(port)),
-        onCancel: () => dispatch(FirmwareDialogActions.hideDialog()),
+        onCancel: port => {
+            dispatch(FirmwareDialogActions.hideDialog());
+            dispatch(SerialPortActions.selectPort(port));
```

Comment:
Do we need to say "proceed without programming"? I would think would be the normal default behavior.